### PR TITLE
Make editorial suggestions to ABS chapter.

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

File: src/chapters/07/main.tex [hunk 1]
```diff
@@ -227,21 +215,25 @@ \section{Solving with Python}\label{sec:solving-with-python}
...
+\end{itemize}
+
 
 The \mintinline{python}{run} method runs the simulation. For each
 \mintinline{python}{n_steps} number of discrete time steps, the city runs the
 method \mintinline{python}{take_turn}.
 In this method, we first create a list of all the houses with households that
 are unhappy with their neighbours; these are put in a random order using the
-\mintinline{python}{random} library; and then working inwards from the ends,
+\mintinline{python}{random} library; and then working inwards from the boundary
```
Comment (geraintpalmer) [inline src/chapters/07/main.tex:236]: I don't think `boundary` is the right word. It the ends of a list, not the boundary of the city. What do you think @drvinceknight ?

Comment (drvinceknight) [reply src/chapters/07/main.tex:236]: How about: `from the inner and outer most elements` ?

File: src/chapters/07/main.tex [hunk 2]
```diff
@@ -398,21 +391,27 @@ \section{Solving with Python}\label{sec:solving-with-python}
...
+In this chapter, a package called \mintinline{R}{R6} will be used here.
+
+For this problem two classes will be built: a
 \mintinline{R}{House} and a \mintinline{R}{City} for them to live in.
 
-Now let's define the \mintinline{R}{City}:
+Now to define the \mintinline{R}{City}: % TODO Add a footnote with something
+                                        % like: Note that for the purposes of
+% pagination of this book no documentation is included in the definition of the
+% class however.
```
Comment (geraintpalmer) [inline src/chapters/07/main.tex:414]: I like this. We can reference back to chapter 1.

File: src/chapters/07/main.tex [hunk 3]
```diff
@@ -462,32 +461,33 @@ \section{Solving with R}\label{sec:solving-with-R}
 \mintinline{R}{run}, \mintinline{R}{take_turn} and
 \mintinline{R}{mean_satisfaction}.
 
-The \mintinline{R}{initialize} method is run at the time the object is first
+The \mintinline{R}{initialise} method is run at the time the object is first
```
Comment (geraintpalmer) [inline src/chapters/07/main.tex:464]: `initialize`

it's a built in R6 function, so spelt the American way

File: src/chapters/07/main.tex (hunk 1 above)
Comment (geraintpalmer) [reply src/chapters/07/main.tex:236]: yeah happy with that

File: src/chapters/07/main.tex (hunk 3 above)
Comment (drvinceknight) [reply src/chapters/07/main.tex]: I am amazed that that's something I changed:

A. I never notice stuff like that.
B. Changing the name of a library is obviously dumb.

What an idiot. 🤦

**Pull request merged**